src/content/en/2021/compression.md

```diff
@@ -1,20 +1,375 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Compression
-description: TODO
+description: Compression chapter of the 2021 Web Almanac covering HTTP compression, algorithms, compression levels, content types, 1st party and 3rd party compression and opportunities.
 authors: [lvandeve, mo271, jyrkialakuijala]
```

Comment:
@lvandeve @mo271 @jyrkialakuijala could each of you add a bio here to the metadata in the form `<author_id>_bio: Foo`? See the [wiki](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) for more info.

@mo271 @jyrkialakuijala you could reuse bios from [last year](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/content/en/2020/compression.md?plain=1#L10-L12) if you'd like.

Reply:
Done, added @mo271 @jyrkialakuijala from last year and a new one for @lvandeve